src/util/unicode.h

```diff
@@ -20,7 +20,6 @@ std::string narrow(const std::wstring &s);
 std::wstring widen(const std::string &s);
 
 std::string utf32_to_utf8(const std::basic_string<unsigned int> &s);
```

Comment:
The above needs to be renamed or even removed (see my suggestion for marking it `static`).